src/services/usergroup.service.ts

```diff
@@ -48,3 +48,15 @@ export const axGetPages = async () => {
     return { success: false, data: [] };
   }
 };
+
+export const axCreateGroup = async ({ ...payload }) => {
```

Comment:
instead of collecting destructuring and spread it's redundant just use `async (payload) => ...`